.test-infra/tools/flaky_test_detection.py

```diff
@@ -65,11 +63,7 @@ def create_github_issue(repo, alert):
     print(f"Body: {body}")
     print(f"Labels: {labels}")
     print("___")
-
-    if READ_ONLY == "true":
-        print("READ_ONLY is true, not creating issue")
-    else:
-        repo.create_issue(title=title, body=body, labels=labels)
+    repo.create_issue(title=title, body=body, labels=labels)
```

Comment:
Can we leave the READ_ONLY setting? Seems useful for local debugging

Reply:
Reverted back